src/Icons/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.23.0
+
+-   Add support for aliases when fetching multiple icons with `Iconify` 
+
```

Comment:
I'd consider this a bug fix (as it worked before i introduced this method), mainly to avoid mixing with the alias term (used two lines later in the changelog haha)

So i would probably not add a changelog entry here to be honest.